src/core/mount.c

```diff
@@ -1638,13 +1637,18 @@ static void mount_sigchld_event(Unit *u, pid_t pid, int code, int status) {
                                 log_unit_warning(u, "Mount still present after %u attempts to unmount, giving up.", m->n_retry_umount);
                                 mount_enter_mounted(m, f);
                         }
+                } else if (f == MOUNT_FAILURE_EXIT_CODE && !m->from_proc_self_mountinfo) {
+                        /* Hmm, umount process spawned by us failed, but the mount disappeared anyway?
+                         * Maybe someone else is trying to unmount at the same time. */
+                        log_unit_notice(u, "Mount still disappeared even though umount failed, ignoring.");
```

Comment:
'still' sounds spurious for me. Also, if `", ignoring."`, I think it is better to downgrade the log level to LOG_DEBUG.

Reply:
(LOG_NOTICE for the same reason, but I'll rephrase the message a bit)